shared/gh/js/controllers/gh.api.util.js

```diff
@@ -149,15 +215,13 @@ define(['exports', 'moment', 'bootstrap-notify'], function(exports, moment) {
             throw new Error('A valid term should be provided');
         }
 
-        // The number of milliseconds in one week
-        var ONE_WEEK = 1000 * 60 * 60 * 24 * 7;
         // Convert the term start and end date to milliseconds
         var termStartDate = new Date(term.start).getTime();
         var termEndDate = new Date(term.end).getTime();
         // Calculate the time difference
         var timeDifference = Math.abs(termEndDate - termStartDate);
         // Convert to weeks and return
-        return Math.floor(timeDifference / ONE_WEEK) + 1;
+        return Math.floor(timeDifference / PERIODS['week']) + 1;
```

Comment:
This is missing the mysterious 0.25 and is using `.floor` instead of `.ceil`. I'm fine with either as long as we're consistent and maybe add some docs that justify the usage.